aip/general/0180.md

```diff
@@ -139,6 +139,30 @@ code, as such changes will be seen as breaking by those users.
 whether a proposed change is likely to break users, and an expansive reading of
 this guidance could ostensibly prevent _any_ change (which is not the intent).
 
+#### Field behavior compatibility
+
+The behavior of field within an API, whether it is required, whether it is only
+presented as output and not accepted as input, or vice versa, is documented with
+the `google.api.field_behavior` annotation as described in [AIP-203][]. Changing
+the value of this annotation can represent a semantic change in the API that is
+percevied as incompatible for existing clients. The following are examples of
+backwards incompatible changes with `google.api.field_behavior`:
+
+* Adding `REQUIRED` to an existing field previously considered `OPTIONAL`
+(implicitly or otherwise).
+* Adding `OUTPUT_ONLY` to an existing field previously accepted as input
+* Removing `OUTPUT_ONLY` from an existing field previously ignored as input
+* Adding `INPUT_ONLY` to an existing field previously emitted as output
+* Adding `IMMUTABLE` to an existing field previously considered mutable
+* Removing `IMMUTABLE` from an existing field previously considered immutable
```

Comment:
Does removing IMMUTABLE cause problems for some clients?

Reply:
Concretely, no, but semantically, I would say yes because a field that was previously considered off limits for changing silently becomes changeable.

This might not actually be an issue because our current guidance states that an API **should** return an error if a change to an immutable field is requested, but I thought to err on the side of caution. WDYT?

Reply:
This is a super common use case for APIs, they tend to release with mostly non-updatable fields and then add update support down the line. This sort of restriction would prevent them from doing that (or at least accurately annotating that fields are immutable) so I would recommend removing it

Reply:
Fair enough! Will do

Reply:
Updated - actually moved it to the "compatible" section